src/components/Issue.tsx

```diff
@@ -1,57 +1,94 @@
 import React, { FC, useState } from 'react';
-
-import Typography from '@material-ui/core/Typography';
+import COLORS from '../utils/colors';
 import TextField from '@material-ui/core/TextField';
-import Button from '@material-ui/core/Button';
 import { SigningProps } from './Props';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
-import Grid from '@material-ui/core/Grid';
 import { JSONEditor } from '@material-did/common';
 import { didDocument } from '../fixtures';
+import styled from 'styled-components';
 import { CircularProgress } from '@material-ui/core';
+import { SignedDocumentRequest } from './../api/index';
+
 const fetch = require('node-fetch');
 
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  font-family: Segoe UI;
+  font-size: 25px;
+  color: ${COLORS.METEORITE};
+  margin-top: 20px;
+`;
+
+const Content = styled.div`
+  display: flex;
+  margin-top: 20px;
+`;
+
+const Title = styled.div`
+  font-size: 2em;
+`;
+
+const Form = styled.form`
+  flex-grow: 1 1 auto;
+  flex-basis: 50%;
+  padding: 10px;
+`;
+
+const SubTitle = styled.div`
+  font-style: italic;
+  font-size: 1.6em;
+  font-weight: bold;
+  margin-bottom: 20px;
+`;
+
+const ContainerDidDoc = styled.div`
+  font-size: 0.3em;
+  color: ${COLORS.METEORITE};
+`;
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      flexGrow: 1,
-      '& .MuiTextField-root': {
-        margin: theme.spacing(1),
-        width: '25ch',
-      },
-    },
-    paper: {
-      padding: theme.spacing(2),
-      textAlign: 'center',
-      color: theme.palette.text.secondary,
-    },
-  }),
-);
+const Button = styled.button`
+  font-size: 0.8em;
+  height: 2em;
+  border: none;
+  border-radius: 10px;
+  color: ${COLORS.METEORITE};
+  margin-top: 2em;
+  background: #f8f8f8;
```

Comment:
All colors write in /utils/colors.ts file. For naming using site http://chir.ag/projects/name-that-color. Must look like  background: ${COLORS.NAME_THIS_COLOR};